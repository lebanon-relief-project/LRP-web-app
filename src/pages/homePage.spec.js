import React from "react";
import { render } from "@testing-library/react";
import HomePage from "./homePage";

describe('the HomePage component', () => {
  
  it("should match the snapshot", () => {
    const { container } = render(<HomePage />);
    expect(container).toMatchSnapshot();
  });
  
})


