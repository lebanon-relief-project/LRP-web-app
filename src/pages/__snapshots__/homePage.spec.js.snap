// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the HomePage component should match the snapshot 1`] = `
<div>
  <div>
    <h2>
      Home Page
    </h2>
  </div>
</div>
`;
